apps/concierge_site/test/lib/helpers/mjml_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule ConciergeSite.MJMLTest do
+  @moduledoc false
+  use ConciergeSite.DataCase, async: true
+  alias ConciergeSite.MJML
+
+  test "works properly with EEx tags" do
```

Comment:
nitpick: This could be wrapped with a `describe` block saying that this is testing `eval_string`.

Reply:
this is using `eval_string`, but the main concept that I wanted to test here is that escaping/unescaping EEx works and that it ultimately gets evaluated properly which feels to me like it is more related to the module as a whole rather than just `eval_string` 🤔 e.g. if we added something like `eval_file` in the future I don't think we should duplicate a similar test under a separate `describe "eval_file/3"` block, if that makes sense?

Reply:
Ah yeah, sure, that makes sense.